go/common/quantity/quantity.go

```diff
@@ -79,6 +83,14 @@ func (q *Quantity) FromInt64(n int64) error {
 	return q.FromBigInt(big.NewInt(n))
 }
 
+// FromUint64 converts from an uint64 to a Quantity.
+func (q *Quantity) FromUint64(n uint64) error {
+	if n > math.MaxInt64 {
```

Comment:
Just handle this? (See: https://golang.org/pkg/math/big/#Int.SetBit)

Reply:
Ah, duh, fixed.